internal/service/adapters.go

```diff
@@ -183,7 +180,7 @@ func NewSchedulerCacheRedis(c config.Config) (ports.SchedulerCache, error) {
 		return nil, fmt.Errorf("failed to initialize Redis scheduler cache: %w", err)
 	}
 
-	return scheduleradapters.NewRedisSchedulerCache(client), nil
+	return scheduler.NewRedisSchedulerCache(client), nil
```

Comment:
```suggestion
	return scheduler.NewRedisCache(client), nil
```

Reply:
Same thing as mentioned before, renaming functions is out of this PR scope